ci/ci.sh

```diff
@@ -138,7 +142,7 @@ run_all(){
 # it gives much better logging breakdown on github
 # the run_all function is only used for manual deployment
 install_go_kubectl_kind
-create_kind_cluster
+create_kind_cluster -v v1.27.3
```

Comment:
why pass in the "-v v1.27.3" here?

Reply:
ah this is for when you manually run the script.

Reply:
since we source the file, it gets overwritten with the current env anyways

Reply:
It also gets me around some of the spell check issues

Reply:
It would be nice to define the Kind test version as a informatively named var somewhere at the top of the file, where it can be found easily later.